ocs_ci/utility/lvmo_utils.py

```diff
@@ -56,3 +62,38 @@ def lvmo_health_check(tries=20, delay=30):
         delay=delay,
         backoff=1,
     )(lvmo_health_check_base)()
+
+
+def get_disks_by_path():
+    oc_obj = OCP()
+    disks = oc_obj.exec_oc_debug_cmd(
+        node=constants.SNO_NODE_NAME, cmd_list=["ls /dev/disk/by-path"]
```

Comment:
You can run `ls /dev/disk/by-path/*`, It will give you the full path. The value `/dev/disk/by-path/*` can be a constant which can be used to list the devices. Similarly, we can make use of other constants `/dev/disk/by-id/*` `/dev/disk/by-partlabel/*` etc as we are supporting other paths as well now.